frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/common/FastForwardMergeProps.kt

```diff
@@ -0,0 +1,8 @@
+package com.virtuslab.gitmachete.frontend.actions.common
+
+data class FastForwardMergeProps(
```

Comment:
Hmm
1. any specific reason for that one being in kotlin?
2. in general, should Kotlin source live under `src/main/java/` or rather `src/main/kotlin/` (does it even make a difference?)